steam.go

```diff
@@ -91,3 +155,120 @@ func GetTF2Stats(steamid string) (*map[string]string, error) {
 
 	return &res, nil
 }
+
+// https://developer.valvesoftware.com/wiki/Steam_Web_API#GetPlayerSummaries_.28v0002.29
+func GetPlayerInfo(steamids []string) (map[string]*PlayerInfo, error) {
```

Comment:
Could you rename this to GetPlayersInfo and create a wrapper GetPlayerInfo function that would take in a single steam id and return a single player info struct?